test/app.test.js

```diff
@@ -89,21 +93,110 @@ describe("app", function() {
                             '8. Child-headed Households'
                         ].join('\n')
                     })
+           //         .check.user.state(function(state) {
+                        //assert
+             //       })
```

Comment:
Do we still need this commented outlines? We should either do something with them or remove them.